### DISCUSSION TITLE
June-September Hiatus

### DISCUSSION THREAD

**Discussion by byteduck** [byteduck/duckOS#64] [Announcements]
Hello all! I'd just like to announce that between the end of June and end of September, I will need to take a hiatus from working on duckOS. Just like last summer, this is due to the NDA/noncompete contract for my internship.

Unfortunately, progress has been slow on duckOS lately due to my inability to work on it over the summer and the breakneck pace of my education during the rest of the year. But, I hope that as I wrap up my education and have less on my to-do list, I should have some more time to spend working on duckOS before I enter either grad school or my career.

Thank you everyone for your contributions so far! I have really enjoyed working on this passion project for the past four years or so. While it hasn't been exactly far-reaching, I've really enjoyed seeing some repeat contributors and I hope to continue to work on duckOS for the foreseeable future. I started working on this in my free time in high school, and I never expected to get this far.

Goodbye for now,
Byteduck 🦆

**Comment by MarcoCicognani** [comment]
Don't worry man, each project has his own era.
Even if we want to have time to continue every idea our mind create...it's not possible.
It's natural to park the projects during the road, but they are always ready to be taken back.

I hope I will have time too to restart the development of my own OS and share ideas and features with your.

Do a good job.
Marco
